blog/_posts/2022-03-02-build-a-resource-catalog.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: "resource-catalog"
+date: 2022-03-03
+author: lizz
+layout: blog
+title: "There's a better way!"
+subtitle: "Use Tilt to build a flexible, browsable resource catalog for your team"
+description: "Use Tilt to build a flexible, browsable resource catalog for your team"
+image: ""
+image_caption: ""
+tags:
+  - tilt
+  - api
+  - ui
+  - groups
+  - labels
+---
+Many developers have far more resources defined in Tilt than they’d actually want to run at once. Do any of these situations seem familiar?
+
+*  Each team requires different subsets of resources to do their day-to-day tasks. 
+*  Work is exploratory and you can’t easily define upfront which resources you’ll need. And you might want to bring up additional resources to test the changes you just made.
+*  Your resources are so CPU-intensive that you only want to run the bare minimum you need.
+
+## A Better Workflow
+
+Tilt already helps you move away from clunky workflows where you need to remember a series of bash commands to run the resources you need. But for managing an active set of resources, we still found people resorting to the command line:
+
+```shell
+# Start Tilt with one resource
+$ tilt up storage
+
+# Oh wait, let's add these other resources I need to work on
+$ tilt args storage frontend glitch object-detector color
+
+# Ahh, my work changed and I need a different set running
+$ tilt args storage frontend object-dectector bounding-box muxer
+```
```

Comment:
does this example get the point across about how cumbersome it can be to swap sets of services? (i'm not sure the comments are necessary.)